src/libstore/binary-cache-store.cc

```diff
@@ -66,6 +67,24 @@ void BinaryCacheStore::init()
                 config.wantMassQuery.setDefault(value == "1");
             } else if (name == "Priority") {
                 config.priority.setDefault(std::stoi(value));
+            } else if (name == "RequiredSignatures") {
```

Comment:
This should probably be named `RequiredSigners` (it's a list of public keys, not a list of signatures).